flake.nix

```
@@ -8,22 +8,40 @@
     };
 
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-
+    naersk.url = "github:nmattia/naersk";
```

You can use the following to avoid pulling in a second nixpkgs:

```suggestion
    naersk = {
      url = "github:nmattia/naersk";
      inputs.nixpkgs.follows = "nixpkgs";
    };
```

The lock file can be rebuilt with something like `nix-shell -I nixpkgs=channel:nixos-unstable -p nixUnstable --run 'nix --experimental-features "nix-command flakes" flake update --recreate-lock-file'`.

---

Nice. Thanks for the tip. Applied.